packages/cozy-harvest-lib/src/services/cccPolicy.js

```diff
@@ -0,0 +1,64 @@
+// @ts-check
+/**
+ * CCC specific policy
+ */
+
+import logger from '../logger'
+
+/**
+ * Check if the given konnector is a client side connector
+ * @param {import('cozy-client/types/types').KonnectorsDoctype} konnector - konnector object
+ * @returns {Boolean}
+ */
+const isCCC = konnector => Boolean(konnector.clientSide)
+
+/**
+ * Get's the launcher in the current environment if any
+ *
+ * @param {Object} win The window object in the current environment
+ * @returns {Object|null}
+ */
+export const getLauncher = ({ win }) =>
+  win?.cozy?.ClientConnectorLauncher || null
```

Comment:
Why do we need to pass the object in params? Can't we just read it from `window` directly?

Reply:
Right, this is not actually needed. No reason to keep it